e2e/package.json

```diff
@@ -18,7 +18,7 @@
     "lodash.mapkeys": "4.6.0",
     "lodash.without": "4.4.0",
     "lodash.xor": "4.5.0",
-    "mattermost-redux": "5.23.0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#8b003ea1aa10bf64b306a006e8a0ed7477924704",
```

Comment:
Good to use fix version but [some of the added types](https://github.com/mattermost/mattermost-webapp/pull/5888/files#diff-f6b9573b81480c95519c79b64ff0e029R7-R11) are not yet available to `5.23.0` nor on the latest npm release version of `5.24.2`. The commit hash here is on master, latest at the moment.

Reply:
Redux hash keeps changing as new commits are added. It will be tough to maintain this. And it's already a nightmare in webapp repo to maintain this. Let's not switch to the commit hash mode. Can we use "master"

Reply:
No, we can't use master. Let's keep an eye and switch once we have the 5.26 version of mattermost-redux.

Reply:
I believe you can use tags instead. Cleaner and easier to see which version until release version is available
https://github.com/mattermost/mattermost-redux/tags

Reply:
The type definition I need is on master or 5.26. Tags were currently up to 5.25 only.

Reply:
Got it. Might help monitor to create a ticket.